src/cli.rs

```diff
@@ -215,6 +215,16 @@ pub struct Cli {
     #[structopt(long, value_name = "FILE", parse(from_os_str))]
     pub cert_key: Option<PathBuf>,
 
+    /// Force a particular TLS version.
+    ///
+    /// "auto" or "ssl2.3" gives the default behavior of negotiating a version
+    /// with the server.
+    #[structopt(long, value_name = "VERSION", parse(from_str = parse_tls_version),
+      possible_values = &["auto", "ssl2.3", "tls1", "tls1.1", "tls1.2", "tls1.3"],
+      default_value = "auto")]
+    // An explicit path to Option disables structopt magic we don't want here
```

Comment:
If you write `Option<...>` structopt makes it optional and expects `parse_tls_version` to return `tls::Version` instead of `Option<tls::Version>`.

But I'm going to change this anyway because of the `default_value` thing.